spec/support/features.rb

```diff
@@ -7,11 +7,23 @@ def in_spec?
 end
 
 def with_env_vars_runner(env)
+  # hold on to old values
   env = env.stringify_keys
```

Comment:
hah you know I didn't even realize it was from tbot until posting that - I'd used it once or twice and just thought it was some rando lib!